spec/controllers/state_file/intake_logins_controller_spec.rb

```diff
@@ -335,6 +335,16 @@
             expect(response).to redirect_to account_locked_portal_client_logins_path
           end
         end
+
+        context "when the client account does not have an ssn" do
```

Comment:
nit: the context block below says "when the intake does not have an ssn" so it would be nice to have consistent descriptions

Reply:
actually now that i've read the context block below these look...the same?

Reply:
hm i think i am responsible for this inconsistent naming but it would be great if the description were "as an unauthenticated intake" instead of "as an unauthenticated client"